sqlbrite/src/main/java/com/squareup/sqlbrite/SqlBrite.java

```diff
@@ -429,6 +430,37 @@ public int update(@NonNull String table, @NonNull ContentValues values,
     return rows;
   }
 
+  /**
+   * Temporarily end the transaction to let other threads run. The transaction is assumed to be
+   * successful so far. Do not call setTransactionSuccessful before calling this. When this
+   * returns a new transaction will have been created but not marked as successful. This assumes
+   * that there are no nested transactions (beginTransaction has only been called once) and will
+   * throw an exception if that is not the case.
+   * @return true if the transaction was yielded
+   *
+   * @see SQLiteDatabase#yieldIfContendedSafely()
+   */
+  public boolean yieldIfContendedSafely() {
+    return getWriteableDatabase().yieldIfContendedSafely();
+  }
+
+  /**
+   * Temporarily end the transaction to let other threads run. The transaction is assumed to be
+   * successful so far. Do not call setTransactionSuccessful before calling this. When this
+   * returns a new transaction will have been created but not marked as successful. This assumes
+   * that there are no nested transactions (beginTransaction has only been called once) and will
+   * throw an exception if that is not the case.
+   * @param sleepAmount if > 0, sleep this long before starting a new transaction if
+   *   the lock was actually yielded. This will allow other background threads to make some
+   *   more progress than they would if we started the transaction immediately.
+   * @return true if the transaction was yielded
+   *
+   * @see SQLiteDatabase#yieldIfContendedSafely(long)
+   */
+  public boolean yieldIfContendedSafely(long sleepAmount, TimeUnit sleepUnit) {
+    return getWriteableDatabase().yieldIfContendedSafely(sleepUnit.toMillis(sleepAmount));
```

Comment:
I believe this is the correct syntax, if you have a better solution please let me know.